web/src/lib/components/asset-viewer/asset-viewer-nav-bar.svelte

```diff
@@ -243,6 +247,13 @@
                 icon={asset.isArchived ? mdiArchiveArrowUpOutline : mdiArchiveArrowDownOutline}
                 text={asset.isArchived ? 'Unarchive' : 'Archive'}
               />
+              {#if showDevUI}
+                <MenuOption
+                  icon={mdiUpload}
+                  on:click={() => openFileUploadDialog({ multiple: false, assetId: asset.id })}
+                  text="Replace with upload"
+                />
+              {/if}
```

Comment:
I think we can probably remove this, but maybe it is useful for testing it in the short term.